pkg/xds/topology/outbound.go

```diff
@@ -6,6 +6,11 @@ import (
 	core_xds "github.com/kumahq/kuma/pkg/core/xds"
 )
 
+const (
+	priorityLocal  = 0
+	priorityRemote = 1
```

Comment:
Please place a short doc that priorityLocal means that all the endpoints in the same zone are marked with the highest priority (0) and will be preferred over endpoints from other zones if zone aware load balancing is on.